.npmignore

```diff
@@ -0,0 +1,11 @@
+.husky/
```

Comment:
this is what I wanted to avoid :) this is well over-specified, meaning that chances are high that as soon as anything changes in the repo, it will either not achieve the desired effect because new artifacts will land in the repo, or even worse that the published package will break because for whatever reason somebody inadvertently included files which are needed for the package in one of the excluded locations.

we're much better off without any ignores in this specific case

Reply:
:D Ok, I mainly intended to exclude the test cases and got a bit carried away. What would you like to add a `*.test.js` ignore pattern?

Reply:
yeah I think it's a safe exclusion